src/pages/MyPage/MyPage.tsx

```diff
@@ -48,6 +50,10 @@ const MyPage = () => {
     gameWrittens: useGameWrittensQuery(),
   };
 
+  const queryisLoading = Object.values(queries).some(
```

Comment:
```suggestion
  const isQueryLoading = Object.values(queries).some(
```
이렇게 네이밍을 가져가는게 더 직관적인 거 같습니닷!

Reply:
수정 완료했습니다~! [18db50e](https://github.com/JECT-Study/Balance-Talk-Frontend/pull/204/commits/18db50ed4c5e1ab3b1136f4dd9e74752838a89fb)